build/commands/lib/logging.js

```diff
@@ -142,9 +142,24 @@ function allPatchStatus(allPatchStatus, patchGroupName) {
       console.log(chalk.green(`${successfulPatches.length} successful patches:`))
       successfulPatches.forEach(logPatchStatus)
     }
+
     if (failedPatches.length) {
-      console.log(chalk.red(`${failedPatches.length} failed patches:`))
-      failedPatches.forEach(logPatchStatus)
+      if (printPatchFailuresInJson) {
+        console.log(
+            chalk.red(`${failedPatches.length} Failed patches json breakdown:`))
+        const patchFailuresOutput = failedPatches.map(({path, patchPath}) => {
+          return {
+            patchPath:
+              patchPath.substring(patchPath.indexOf('brave/patches')),
+                  path: path
```

Comment:
you don't want the failure reason?

Reply:
For now it is not needed. It may be interesting in the future, but if that becomes the case I might take another iteration on it.